### PR TITLE
refactor(v2): cleanup console output

### PR DESCRIPTION
**Pull request by lex111**
## Motivation

Attempting to tidy up any output in the console that Docusaurus codebase produces.

Main points:

- Avoid using "programmatic" constructs like `[]` or `=`, and use a more user-friendly style of spelling instead (i.e. `=[',']` -> `","`)
- Using double quotes for values, fields, etc.
- Adding a dot at the end of console message (except CLI options, no needed put dot in description of CLI options)
- Making correct spelling of some words (eg. NodeJS -> Node.js)

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

Snapshots?

## Related PRs

(If this PR adds or changes functionality, please take some time to update the docs at https://github.com/facebook/docusaurus, and link to your PR here.)

### CODE REVIEW

File: packages/docusaurus-init/src/index.ts [hunk 1]
```diff
@@ -182,23 +182,26 @@ export default async function init(
...
   console.log();
-  console.log('We suggest that you begin by typing:');
+  console.log('We recommend that you begin by typing:');
   console.log();
   console.log(chalk.cyan('  cd'), cdpath);
   console.log(`  ${chalk.cyan(`${pkgManager} start`)}`);
 
   console.log();
-  console.log('Happy hacking!');
+  console.log('Happy building awesome websites!');
```
Comment (lex111) [inline packages/docusaurus-init/src/index.ts:206]: It's an attempt to distinguish from the [CRA](https://github.com/facebook/create-react-app/blob/fddce8a9e21bf68f37054586deb0c8636a45f50b/packages/react-scripts/scripts/init.js#L408), I think it's fine to have an original slogan.

**Comment by lex111** [comment]
I don't really understand why the tests do not pass successfully, because snapshots do not differ :man_shrugging: 

image

**Review by slorber — APPROVED** [review]
Thanks 👍  LGTM

**Comment by slorber** [comment]
> I don't really understand why the tests do not pass successfully, because snapshots do not differ 🤷‍♂️

Not sure what happened but Jest doesn't seem to like my test factorization much so I removed it and restored a bit of duplication, now the snapshots pass and can be updated

**Pull request merged**